### PR TITLE
feat(build): cstor repo migration to mayadata-io org

### PR DESCRIPTION
**Pull request by pawanpraka1**
Signed-off-by: Pawan <pawan@mayadata.io>

### CODE REVIEW

File: docker/build.sh [hunk 1]
```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CSTOR_ORG="mayadata-io"
```
Comment (mittachaitu) [inline docker/build.sh]: nit: We can pass it as an argument to script

**Review by mittachaitu — APPROVED** [review]
LGTM

File: docker/build.sh (hunk 1 above)
Comment (akhilerm) [inline docker/build.sh]: Can we have this also passed via the github action. same like did in other files. Or set the value to `mayadata-io` only if the CSTOR_ORG variable is unset.

**Review by akhilerm — COMMENTED** [review]
@pawanpraka1 Has one minor comment

File: docker/build.sh (hunk 1 above)
Comment (pawanpraka1) [reply docker/build.sh]: making it fail will be more assertive. made the changes.

**Review by mittachaitu — APPROVED** [review]
LGTM

**Review by akhilerm — APPROVED** [review]
LGTM

**Pull request merged**